src/lib.rs

```diff
@@ -147,9 +147,7 @@ pub fn test(_attr: TokenStream, item: TokenStream) -> TokenStream {
 ///
 /// #[async_attributes::bench]
 /// async fn bench_1(b: &mut test::Bencher) {
-///     b.iter(|| {
-///         println!("hello world");
-///     })
+///     println!("hello world");
```

Comment:
Aren't we missing `b.iter(...)` here?

Reply:
This is now part of the macro again, unfortunately.